src/types/models/ObjectModel.jl

```diff
@@ -35,18 +32,12 @@ keys(model.reactions)
 # Fields
 $(TYPEDFIELDS)
 """
-mutable struct ObjectModel <: AbstractMetabolicModel
+Base.@kwdef mutable struct ObjectModel <: AbstractMetabolicModel
     id::String
-    reactions::OrderedDict{String,Reaction}
-    metabolites::OrderedDict{String,Metabolite}
-    genes::OrderedDict{String,Gene}
-
-    ObjectModel(
-        id = "";
-        reactions = OrderedDict{String,Reaction}(),
-        metabolites = OrderedDict{String,Metabolite}(),
-        genes = OrderedDict{String,Gene}(),
-    ) = new(id, reactions, metabolites, genes)
+    reactions::OrderedDict{String,Reaction} = OrderedDict{String,Reaction}()
+    metabolites::OrderedDict{String,Metabolite} = OrderedDict{String,Metabolite}()
+    genes::OrderedDict{String,Gene} = OrderedDict{String,Gene}()
+    objective::Dict{String,Float64} = Dict{String,Float64}()
 end
```

Comment:
btw you may want to check SBML source -- iirc you can actually add docstrings to fields which get rendered as documentation in $(TYPEDFIELDS)

Reply:
see e.g. here:
https://github.com/LCSB-BioCore/PikaParser.jl/blob/master/src/structs.jl#L311
and rendering here:
https://lcsb-biocore.github.io/PikaParser.jl/stable/reference/#PikaParser.ParserState

Reply:
ahhh, okay, will make OM prettier